src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+- Authenticating both EL and CL clients with a shared JWT secret so they can securely communicate with one another.
+
+Not completing the above items in time for The Merge will result in your node appearing to be "offline" until both layers are synced and authenticated.
+
+Node operators can also check out the [Merge Readiness Checklist](https://launchpad.ethereum.org/en/merge-readiness/) on the Staking Launchpad for more information, as much of the details apply to all node operators.
+</ExpandableCard>
+
+<ExpandableCard
+title="Dapp and smart contract developers"
+contentPreview="The Merge has been designed to have minimal impact on smart contract and dapp developers, but there are a few small things devs may want to be aware of heading into The Merge."
+id="developers">
+These changes relate to <br/><br/>
+
+- block structure
+- slot/block timing
+- opcode changes
+- sources of on-chain randomness
+- concept of _safe head_ and _finalized blocks_
+
+For more information, check out this blog post by Tim Beiko on [How The Merge Impacts Ethereum’s Application Layer](https://blog.ethereum.org/2021/11/29/how-the-merge-impacts-app-layer/).
+</ExpandableCard>
+
+### Users and holders {#users-holders}
 
-Imagine Ethereum is a space ship that isn’t quite ready for an interstellar voyage. With the Beacon Chain the community has built a new engine and a hardened hull. When it’s time, the current ship will dock with this new system, merging into one ship, ready to put in some serious lightyears and take on the universe.
+<strong>You do not need to anything to protect your funds entering The Merge.</strong><br/><br/>
 
-## Merging with Mainnet {#docking-mainnet}
+<em>This bears repeating</em>: As a user or holder of ETH or any other digital asset on Ethereum, <strong>you do not need to do anything with your funds or wallet before The Merge.</strong>
 
-When ready, Ethereum Mainnet will "merge" with the Beacon Chain, becoming its own shard which uses proof-of-stake instead of [proof-of-work](/developers/docs/consensus-mechanisms/pow/).
+Despite swapping out proof-of-work, the entire history of Ethereum since genesis remains intact and unaltered after the transition to proof-of-stake. Any funds being protected by your own wallet before The Merge will still be accessible in the same manner after The Merge. No action is required to "upgrade" on your part.
 
-Mainnet will bring the ability to run smart contracts into the proof-of-stake system, plus the full history and current state of Ethereum, to ensure that the transition is smooth for all ETH holders and users.
+As we approach The Merge of Ethereum Mainnet, **you should be on high alert for scams trying to take advantage of users during this transition.** Do not send your ETH anywhere in attempt to "upgrade to ETH2." There is no "ETH2" token, and there is nothing more you need to do for your funds to remain safe.
+
+[More on Ethereum security](/security/#eth2-token-scam)
```

Comment:
This whole section should be a fourth expandable card IMO, users/holders are just another group who need to be ready for the merge (even though the 'steps' to be ready are less)

Reply:
Had originally placed them in there, but @nloureiro and I discussed this, and came to this for a few reasons.

First it visually didn't look great since the page ended up mostly as cards. Mainly though, we want to reduce the burden for day-to-day users from seeing this information, which keeping it in a collapsed card would require them to click in.

Doing this for every category made the page way too long, and also those personas apply to a smaller population.

Reply:
I agree w/ @minimalsm's initial point - making this one of the collapsable cards seems a better fit here.

To start, the initial text up top:

> For everyday users, holders, or non-node-operating stakers there is nothing you need to do, but a few things you should be aware of heading into The Merge. More on this below.

Seems less obvious/apparently because it doesn't stand out like the cards below. It's probably the most important text on the page. I feel like it's currently too underwhelming.

Then having a link that skips down to a section at the end (vs. just expanding a card) seems less intuitive.

I can understand the goal (reducing burden) but I don't think this current solution achieves that well. If not the first collapsable card, I would have entire "Users and holders" section above the others instead of breaking it up.

Reply:
@nloureiro ^

Reply:
I think moving the section up would be the best compromise. I disagree with the user section being collapsed.